llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -33,6 +34,7 @@ def __init__(
         timeout: Optional[float] = 10.0,
         disable_validation: bool = False,
         verbose: bool = False,
+        service_manager: ServiceManager = ServiceManager(),
```

Comment:
I imagine llama-agents would want to wrap services into some http-based RPC system, so that the step would keep calling `service.run()` but under the hood that would translate in something llama-agent specific. Whether this should be user-facing or not, it entirely depends on the type of user, like a llama-agent developer would want to use this, a workflow user wouldn't probably want to know about it.

Reply:
Change of mind, let me try by making the `ServiceManager` more prominent.